messaging.tests/Integration/BundlesControllerTests.cs

```diff
@@ -633,6 +633,38 @@ public async void GetWithInvalidJurisdictionGetsError()
             Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
         }
 
+        [Fact]
+        public async void PostWithNonMatchingJurisdictionsError()
+        {
+            // Clear any messages in the database for a clean test
+            DatabaseHelper.ResetDatabase(_context);
+
+            // The jurisdiction ID of the input record submission is 'NY', which should not work with a 'PA' endpoint parameter.
+            string jurisdictionParameter = "PA";
+
+            // Create a new Death Record
+            DeathRecordSubmissionMessage recordSubmission = BaseMessage.Parse<DeathRecordSubmissionMessage>(FixtureStream("fixtures/json/DeathRecordSubmissionMessage.json"));
+
+            // Submit that Death Record
+            HttpResponseMessage createSubmissionMessage = await JsonResponseHelpers.PostJsonAsync(_client, $"/{jurisdictionParameter}/Bundle", recordSubmission.ToJson());
+            Assert.Equal(HttpStatusCode.BadRequest, createSubmissionMessage.StatusCode);
+        }
+
+        [Fact]
+        public async void PostBatchWithNonMatchingJurisdictionsError()
```

Comment:
The Batch return code should actually be a 200 in this case and the individual responses within the batch response should have the 400 bad request response. There may be some other batch tests that show how to check this.

Reply:
Both comments should be resolved